sdk/devcenter/azure-developer-devcenter/README.md

```diff
@@ -0,0 +1,156 @@
+[azure-identity][azure_identity_credentials] library.
+
+To authenticate with AAD, you must first [pip][pip] install [`azure-identity`][azure_identity_pip]
+
+After setup, you can choose which type of [credential][azure_identity_credentials] from azure.identity to use.
+As an example, [DefaultAzureCredential][default_azure_credential] can be used to authenticate the client:
+
+Set the values of the client ID, tenant ID, and client secret of the AAD application as environment variables:
+`AZURE_CLIENT_ID`, `AZURE_TENANT_ID`, `AZURE_CLIENT_SECRET`
+
+Use the returned token credential to authenticate the client:
+
+```python
+>>> import os
+>>> from azure.developer.devcenter import DevCenterClient
+>>> from azure.identity import DefaultAzureCredential
+>>> tenant_id = os.environ['AZURE_TENANT_ID']
+>>> client = DevCenterClient(tenant_id: tenant_id, dev_center: "my_dev_center", credential=DefaultAzureCredential())
+```
+
+## Examples
+
+### Dev Box Management
+```python
+>>> import os
+>>> from azure.developer.devcenter import DevCenterClient
+>>> from azure.identity import DefaultAzureCredential
+>>> from azure.core.exceptions import HttpResponseError
+>>> tenant_id = os.environ['AZURE_TENANT_ID']
+>>> client = DevCenterClient(tenant_id: tenant_id, dev_center: "my_dev_center", credential=DefaultAzureCredential())
+>>> try:
+        # Fetch control plane resource dependencies
+        projects = list(client.dev_center.list_projects(top=1))
+        target_project_name = projects[0]['name']
+
+        pools = list(client.dev_boxes.list_pools(target_project_name, top=1))
+        target_pool_name = pools[0]['name']
+
+        # Stand up a new dev box
+        create_response = client.dev_boxes.begin_create_dev_box(target_project_name, "Test_DevBox", {"poolName": target_pool_name})
+        devbox_result = create_response.result()
+
+        LOG.info(f"Provisioned dev box with status {devbox_result['provisioningState']}.")
+
+        # Connect to the provisioned dev box
+        remote_connection_response = client.dev_boxes.get_remote_connection(target_project_name, "Test_DevBox")
+        LOG.info(f"Connect to the dev box using web URL {remote_connection_response['webUrl']}")
+
+        # Tear down the dev box when finished
+        delete_response = client.dev_boxes.begin_delete_dev_box(target_project_name, "Test_DevBox")
+        delete_response.wait()
+        LOG.info("Deleted dev box successfully.")
+    except HttpResponseError as e:
+        print('service responds error: {}'.format(e.response.json()))
+
+```
+
+### Environment Management
+```python
+>>> import os
+>>> from azure.developer.devcenter import DevCenterClient
+>>> from azure.identity import DefaultAzureCredential
+>>> from azure.core.exceptions import HttpResponseError
+>>> tenant_id = os.environ['AZURE_TENANT_ID']
+>>> client = DevCenterClient(tenant_id: tenant_id, dev_center: "my_dev_center", credential=DefaultAzureCredential())
```

Comment:
same with https://github.com/Azure/azure-sdk-for-python/pull/26696/files#r998893351